'solc --version' running
Traceback (most recent call last):
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\platform\solc.py", line 380, in get_version
    with subprocess.Popen(
         ~~~~~~~~~~~~~~~~^
        cmd,
        ^^^^
    ...<3 lines>...
        executable=shutil.which(cmd[0]),
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ) as process:
    ^
  File "C:\Python313\Lib\subprocess.py", line 1036, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
    ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
                        pass_fds, cwd, env,
                        ^^^^^^^^^^^^^^^^^^^
    ...<5 lines>...
                        gid, gids, uid, umask,
                        ^^^^^^^^^^^^^^^^^^^^^^
                        start_new_session, process_group)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python313\Lib\subprocess.py", line 1548, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
                       ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
                             # no special security
                             ^^^^^^^^^^^^^^^^^^^^^
    ...<4 lines>...
                             cwd,
                             ^^^^
                             startupinfo)
                             ^^^^^^^^^^^^
FileNotFoundError: [WinError 2] The system cannot find the file specified

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\Scripts\slither.exe\__main__.py", line 7, in <module>
    sys.exit(main())
             ~~~~^^
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\slither\__main__.py", line 776, in main
    main_impl(all_detector_classes=detectors, all_printer_classes=printers)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\slither\__main__.py", line 882, in main_impl
    ) = process_all(filename, args, detector_classes, printer_classes)
        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\slither\__main__.py", line 96, in process_all
    compilations = compile_all(target, **vars(args))
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\crytic_compile.py", line 722, in compile_all
    compilations.append(CryticCompile(target, **kwargs))
                        ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\crytic_compile.py", line 211, in __init__
    self._compile(**kwargs)
    ~~~~~~~~~~~~~^^^^^^^^^^
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\crytic_compile.py", line 633, in _compile
    self._platform.compile(self, **kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\platform\solc.py", line 150, in compile
    targets_json = _get_targets_json(compilation_unit, self._target, **kwargs)
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\platform\solc.py", line 279, in _get_targets_json
    return _run_solc(
        compilation_unit,
    ...<6 lines>...
        force_legacy_json=force_legacy_json,
    )
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\platform\solc.py", line 496, in _run_solc
    compiler="solc", version=get_version(solc, env), optimized=is_optimized(solc_arguments)
                             ~~~~~~~~~~~^^^^^^^^^^^
  File "C:\Users\JInfi\AppData\Roaming\Python\Python313\site-packages\crytic_compile\platform\solc.py", line 400, in get_version
    raise InvalidCompilation(error)
crytic_compile.platform.exceptions.InvalidCompilation: [WinError 2] The system cannot find the file specified
